tests/stdlib/trstgen.nim

```diff
@@ -857,9 +903,11 @@ Test1
 
       Ref. [#note]_
       """
-    # TODO: find out hot to configure proper exception instead of defect
-    expect(AssertionDefect):
-      let output5 = input5.toHtml
+    var error5 = new string
```

Comment:
nit: instead of having to bump up a counter, use blocks so each test is independent in terms of variables in scope (makes it easier to move them around, add some without messing re-ordering, etc)

Reply:
Agree. I suggest to do a follow-up PR for `trstgen.nim`:

- split to separate tests as you said
- use `dedent` in this test everywhere

wdyt?

Reply:
yes, in a different PR is ok, to make progress faster and reviewing easier :)